{
  "_args": [
    [
      "is-retry-allowed@1.2.0",
      "/home/runner/work/add-and-commit/add-and-commit"
    ]
  ],
  "_development": true,
  "_from": "is-retry-allowed@1.2.0",
  "_id": "is-retry-allowed@1.2.0",
  "_inBundle": false,
  "_integrity": "sha512-RUbUeKwvm3XG2VYamhJL1xFktgjvPzL0Hq8C+6yrWIswDy3BIXGqCxhxkc30N9jqK311gVU137K8Ei55/zVJRg==",
  "_location": "/is-retry-allowed",
  "_phantomChildren": {},
  "_requested": {
    "type": "version",
    "registry": true,
    "raw": "is-retry-allowed@1.2.0",
    "name": "is-retry-allowed",
    "escapedName": "is-retry-allowed",
    "rawSpec": "1.2.0",
    "saveSpec": null,
    "fetchSpec": "1.2.0"
  },
  "_requiredBy": [
    "/got"
  ],
  "_resolved": "https://registry.npmjs.org/is-retry-allowed/-/is-retry-allowed-1.2.0.tgz",
  "_spec": "1.2.0",
  "_where": "/home/runner/work/add-and-commit/add-and-commit",
  "author": {
    "name": "Vsevolod Strukchinsky",
    "email": "floatdrop@gmail.com",
    "url": "github.com/floatdrop"
  },
  "bugs": {
    "url": "https://github.com/floatdrop/is-retry-allowed/issues"
  },
  "dependencies": {},
  "description": "Is retry allowed for Error?",
  "devDependencies": {
    "ava": "^0.8.0",
    "xo": "^0.12.1"
  },
  "engines": {
    "node": ">=0.10.0"
  },
  "files": [
    "index.js"
  ],
  "homepage": "https://github.com/floatdrop/is-retry-allowed#readme",
  "keywords": [],
  "license": "MIT",
  "name": "is-retry-allowed",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/floatdrop/is-retry-allowed.git"
  },
  "scripts": {
    "test": "xo && ava"
  },
  "version": "1.2.0"
}
